src/main/resources/twilio-node/imports.mustache

```diff
@@ -7,3 +7,4 @@ import { {{key}} } from "{{apiVersionPath}}/../../../{{value}}";
 {{/vendorExtensions.x-import}}
 {{/vars}}
 {{/responseModel}}
+{{#hasTwimlType}}import TwiML from "{{apiVersionPath}}/../../twiml/TwiML";{{/hasTwimlType}}
```

Comment:
Can this be replaced with updating the `library` section of `node.json` to include an entry for `twiml`?

Reply:
Could we import them all once and then remove unused with tooling?

Reply:
Added [function](https://github.com/twilio/twilio-oai-generator/pull/391/files#diff-7e15ccacf5f960109accbd6a4cca462305184eb56f17bd54ff9de6bbb6ef8475R53) for it